code/nacl-box/Hacl.NaCl.fst

```diff
@@ -99,6 +129,11 @@ let crypto_secretbox_open_easy m c clen n k =
   Hacl.Impl.SecretBox.secretbox_open_easy (clen -! 16ul) m k n c
 
 
+[@@ Comment "Compute a shared secret key given a public key and secret key.
```

Comment:
Should we specify the length of `pk` and `sk`?

Reply:
Added in later fixups.